### ISSUE TITLE
Table input freezes with "High Precision Mouse" enabled

### ISSUE THREAD

**Issue by peppy** [ppy/osu#19960]
Not sure it's out fault, but something regressed recently causing this to happen to more users. Might be something framework, SDL, OTD or external.

See:
https://github.com/ppy/osu/discussions/19588
https://github.com/ppy/osu/discussions/19655
https://github.com/ppy/osu/discussions/19944
and probably more..

### Discussed in https://github.com/ppy/osu/discussions/19914

<div type='discussions-op-text'>

<sup>Originally posted by **Boo-BooLazer** August 23, 2022</sup>
I posted a discussion post about a month ago pertaining to this same issue. Am still having the exact same problems and I also noticed that another user is having similar issues to me as well. I don't believe that this problem is on my end as I have been using lazer for years and have never had this problem until recently. Here's the link to the original post, https://github.com/ppy/osu/discussions/19655#discussion-4289676 
Here is a link to the post I mentioned above, 
https://github.com/ppy/osu/discussions/19588#discussion-4281558

For your information, I am currently on version 2022.821.0

Here are my current logs, but you can find my original logs in the original post.

[performance.log](https://github.com/ppy/osu/files/9396918/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9396919/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9396920/updater.log)
[database.log](https://github.com/ppy/osu/files/9396921/database.log)
[network.log](https://github.com/ppy/osu/files/9396922/network.log)

EDIT: I forgot to mention that I just reinstalled Wacom's most updated drivers for my pen tablet. Here is the website to download, https://www.wacom.com/en-us/support/product-support/drivers. My tablet is the CTH-460.</div>

**Comment by approachcircle** [comment]
a somewhat decent fix for this issue is here: #19186, but it isn't perfect considering you have to try and eyeball the tablet area between OpenTabletDriver in lazer, and the external driver's tablet area.

i was going to make a comment about how all of these issues seem to be happening to wacom tablets only (mine included), but then i came across one of the issues that occurred with a huion tablet.

**Comment by Ezbaze** [comment]
Disabling High Preccision Mode fixes it, which is kind of sad as the ratio lock in osu for tablet area scalling is very usefull and waccom doesn't have it for whatever reason

**Comment by approachcircle** [comment]
has there been any progression made on this issue yet? it's the sole reason i haven't switched to lazer yet now that pp can be gained

**Comment by Walavouchey** [comment]
no, any progress would be seen here. you can click "subscribe" on the thread to receive notifications for whenever this gets picked up

**Comment by JonathanSilvestre** [comment]
Has been any progress made on this issue yet? I want to play lazer with my pen tablet.

**Comment by YuunaSol** [comment]
This needs to be fixed ASAP! I literally cannot play the game on laser because I can't get the right sensitivity. I do not play full area and strongly dislike how it feels, I like how the "mouse" mode on tablet feels. I'm legit super forward thinking and appreciative of the progress made on osu!lazer, I am looking forward to the future. However, this needs to be much MUCH higher priority. Sensitivity is literally something that can make or break players, it would be almost as bad as not being able to adjust the audio offset. I want to be on laser so bad, but this is really not okay, especially since the issue has persisted for so long now.

Also, on a side note "lazer" isn't even a word. It's a misspelling of "laser" and it causes me a lot of confusion when searching for it lol. Laser is an acronym.

**Comment by Ficatino** [comment]
Today I tested this issue on the same hardware but on a different system, and I don't have the issue on my new system, but I have it on my old one. So I can confirm that the issue has nothing to do with osu lazer itself, but may be a conflict caused by some software.

**Comment by TheStargater94** [comment]
I got it working by installing Open Tablet Driver (https://opentabletdriver.net/).  Since i have a Huion 420 it didn't work out of the box. It was not recognized by OpenTabletDriver. As seen on the list which tablets are supported (https://opentabletdriver.net/Tablets) it said "Windows: Requires Zadig’s WinUSB to be installed on interface 0". I got the .exe from here "https://github.com/pbatard/libwdi/releases". Started Zadig and clicked "Options" and "List All Devices". Selected the tablet in my case "420 (Interface 0)" and clicked replace Driver. A restart of the pc was needed. After that i could start OpenTabletDriver and the tablet was recognized. I didn't change the tablet area and minimized it. Now i can start osu and turn on "high precision mouse" and adjust the cursor sensitivity like i was used to before. Didnt have that problem with the old osu version. Happened since i switched to osu! lazer. Something i did before too was to install the driver from the manufacturer called "TabletDriver_3Expresskey_0Softkey.exe". But i uninstalled it after because i was not was not sure if it would disturb OpenTabletDriver. Glad i got the Huion 420 working on Windows 11.

**Comment by peppy** [comment]
This should be fixed with recent changes.
